libcudacxx/include/cuda/__memory_resource/cuda_managed_memory_resource.h

```diff
@@ -177,6 +177,9 @@ class cuda_managed_memory_resource
 static_assert(resource_with<cuda_managed_memory_resource, device_accessible>, "");
 static_assert(resource_with<cuda_managed_memory_resource, host_accessible>, "");
 
+//! @brief globally available instance of a ``cuda_managed_memory_resource`` with flags set to ``cudaMemAttachGlobal``
```

Comment:
Question: why double back quotes?

Reply:
That is coming from me writing too much rst docs.

In rst we need to use double quotes, but in "pure" doxygen we only need one